packages/app-page-builder/src/editor/components/Element.tsx

```diff
@@ -100,6 +100,19 @@ const Element = (props: ElementProps) => {
         return null;
     }
 
+    let renderedPlugin = null
+    try {
+        renderedPlugin = plugin.render({ element })
+    } catch(err) {
+        renderedPlugin = (
+            <div>
+                <div>{err.message}</div>
```

Comment:
Could you spare some time and style this error a little bit better? See how CRA styles their error overlay. Something along those lines would be great. Just to have a clear error message, and maybe skip the stack output, since you're console.logging it, and simply output a message like `To see the error stack, open dev tools ...`. It will make the output less messy in a page full of elements.

Reply:
Fingers crossed that this will do 💈